examples/benchmarks/train_resnet_imagenet1k.py

```diff
@@ -0,0 +1,320 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script demonstrates how to train torchvision ResNet architectures on ImageNet using Composer's Trainer. The
+default settings replicate our baseline results. Below illustrates how some arguments can be set to replicate
+our fastest ResNet recipe, our highest accuracy ResNet recipe, and several other settings. This script contains several
+additional features such as saving checkpoints, resume training from a saved checkpoint, speed monitor to track
+training time and throughput, and a learning rate monitor to track learning rate.
```

Comment:
```suggestion
"""
This script demonstrates how to train torchvision ResNet architectures on ImageNet using Composer's Trainer. The
default settings replicate our baseline results. Below illustrates how some arguments can be set to replicate
our fastest ResNet recipe, our highest accuracy ResNet recipe, and several other settings. This script contains several
additional features such as saving checkpoints, resuming training from a saved checkpoint, logging training speed over time, and a logging learning rate over time.
"""
```

Ignore the closing """ at the end; I just put that there so the diff figures out it's a docstring and becomes legible.